README.md

```diff
@@ -17,14 +17,14 @@ Node doesn't allow using `.js` extension for both `esm` and `commonjs` files in
 
 When running this package, you will get an error, depending on it's `type` field:
 
-- run `node cjs/test-esm.js` to get `require() of ES Module` error (if package has no `type` or `"type": "commonjs"`)
+- run `node cjs/test-esm.js` to get `require() of ES modules is not supported.` error (if package has no `type` or `"type": "commonjs"`)
 
-  - which means you can only use `import` syntax with this module in Node
-  - when running `node cjs/test-cjs.mjs`, you will not get an error
+  - which means you can only use `require` syntax with this module in Node
```

Comment:
How can this be true, if in this example we are requiring module and getting the error? 

You fix implies it won't throw

Reply:
Okay, there is some midunderstanding.

> run `node cjs/test-esm.js` to get `require() of ES Module` error (if package has no `type` or `"type": "commonjs"`)

In the module of "run node `cjs/test-esm.js`", package.json has `"type": "module"` => can only use `import`.

In my previous understanding, in the module of "if package has no `type` or `"type": "commonjs"`" => can only use `require`.